website/pages/docs/plugins/sources/k8s/overview.mdx

```diff
@@ -16,21 +16,25 @@ The K8s Source plugin for CloudQuery extracts configuration from a variety of K8
 
 ## Authentication
 
-Similar to how `kubectl` works, `cloudquery` depends on a Kubernetes configuration file to connect to a Kubernetes cluster and `sync` its information. By default, `cloudquery` uses the default Kubernetes configuration
-file (`~/.kube/config`). You can also specify a different configuration by setting the `KUBECONFIG` environment variable before running `cloudquery sync`.
+Similar to how `kubectl` works, `cloudquery` depends on a Kubernetes configuration file to connect to a Kubernetes cluster and `sync` its information.
+By default, `cloudquery` uses the default Kubernetes configuration file (`~/.kube/config`).
+You can also specify a different configuration by setting the `KUBECONFIG` environment variable before running `cloudquery sync`.
 
 ```bash copy
-export KUBECONFIG=<PATH_TO_YOUR_CONFIG_FILE>
+export KUBECONFIG="<PATH_TO_YOUR_CONFIG_FILE>"
 ```
 
-If `cloudquery` is running in a pod of the Kubernetes cluster whose information will be synced, the K8s service account can be used for authentication directly.
-First, create a cluster role named `cloudquery-cluster-read` with all get and list privileges of all resources.
-Then create a cluster role binding for the cluster role and the service account for the `cloudquery` pod to use.
+### Kubernetes Service Account
+
+When `cloudquery` is syncing the Kubernetes cluster information where it is running in a pod,
```

Comment:
I like using `if` here more.   For example:

If `cloudquery` is running in a pod of the Kubernetes cluster, the Kubernetes Service Account can be used for direct authentication.  To use the Kubernetes Service Account for direct authentication, a cluster role with all `get` and `list` privileges will need to be used.

The below command creates a new cluster role with `get` and `list` privileges.